packages/hash/frontend/src/shared/layout/layout-with-sidebar/sidebar.tsx

```diff
@@ -80,7 +81,7 @@ export const PageSidebar: VoidFunctionComponent = () => {
         href="/"
         tooltipTitle="Pages you’ve recently visited"
       /> */}
-      <Box sx={{ mb: 1.5 }} />
+      {/* <Box sx={{ mb: 1 }} /> */}
```

Comment:
```suggestion
      {/* <Box sx={{ mb: 1 }} /> */}
```
We can go ahead deleting this

Reply:
👍 All done